### PR TITLE
[API 2.0: doc] transfer from paddle.fluid.layers.assign() into creation.py

### PR DESCRIPTION
**Pull request by vslyu**
### PR types

Others
### PR changes

APIs
### Describe

- assign
image

### CODE REVIEW

File: python/paddle/tensor/creation.py [hunk 1]
```diff
@@ -1106,3 +1107,77 @@ def empty_like(x, dtype=None, name=None):
         stop_gradient=True)
     out.stop_gradient = True
     return out
+
+
+def assign(x, output=None):
+    """
+ 
+ 
+    The OP copies the :attr:`input` to the :attr:`output`.
```
Comment (jzhang533) [inline python/paddle/tensor/creation.py]: input 应为 x

**Review by jzhang533 — APPROVED** [review]
lgtm

**Review by lanxianghit — APPROVED** [review]
该API没有添加动态图逻辑，会影响性能，需要在下个版本进行添加

**Review by zhiqiu — APPROVED** [review]
Please add dygraph branch in the future.

**Review by kolinwei — APPROVED** [review]

**Pull request merged**